wordle_main.py

```diff
@@ -1,130 +1,132 @@
 
 
-# CLASSES
-
-# I think I would design this a bit differently. I would create a wrapper class for the entire game, and use methods to perform actions.
-#
-# class WordleGame:
-#     def create_row():
-#         ...
-#     def create_board():
-#         ...
-#     def run_game():
-#         ...
-#     the rest of the functions from 'MESSAGES' would live in here too 
-
-
-
-# i don't love calling these things 'Squares' but i don't really have a better suggestion right now...
-# i also think this entire class can just be:
-
-# def colorize_square(letter, color='white'):
-    # return colored(f'[{letter}]', color)
-
-class Square:
-    def __init__(self, letter, color='white'):
-        self.letter = letter
-        self.color = color
-
-    # not used
-    def change_color(self, new_color):
-        """change instance color from default white to given color"""
-        self.color = new_color
-        return self.color
-
-    def display_square(self):
-        # doesn't look like this exists
-        print(self.create_column())
+class GuessLengthException(Exception):
+    """Raise if len(guess) != 5."""
+    pass
 
-    def create_formatted_square(self):
-        """returns a complete column to be used in row class"""
-        # one-liner:
-        # return colored(f'[{self.letter}]', self.color)
-        letter = self.letter
-        color = self.color
-        formatted_square = colored(f'[{letter}]', color)
-        return formatted_square
 
+class InvalidWordException(Exception):
+    """Raises if word not in possible_guesses."""
+    pass
 
-# this class can just be one method create_row() on the main game class, no real benefit of using a class here.
-class Row:
-    def __init__(self, squares):
-        self.squares = squares
 
-    def create_formatted_row(self):
-        # one liner here:
-        # return ''.join([s for s in self.squares])
-        row = []
-        for square in self.squares:
-            row.append(square)
-        return ''.join(row)
+class WordleGame:
+    squares = []
+    rows = [None] * 6
+    player_name = None
 
+    def create_row(self):
+        """Formats each letter from user's guess and returns a combined displayable row"""
+        return ''.join([s for s in self.squares])
 
-# same here, just create_board() method
-class Board:
-    def __init__(self, rows):
-        self.rows = rows
-
-    def create_board(self):
-        board = []
+    def display_board(self):
         for row in self.rows:
-            # just `if row:` (don't need `is None`)
-            if row is None:
-                board.append('[ ][ ][ ][ ][ ]')
+            if row is not None:
+                print(row)
+            else:
+                print('[ ][ ][ ][ ][ ]')
+
+    def run_game(self):
+
+        answer = random.choice(possible_answers)
+        letters_remaining = ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M',
+                             'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z'
```

Comment:
The issue with this is the possible_answers variable which is storing a list of words from the txt file. I can't use import_word_list() function within the class since it is declared outside. Any thoughts?